src/View/Helper/FormHelper.php

```diff
@@ -420,6 +420,7 @@ public function create($model = null, array $options = [])
         $htmlAttributes += $options;
 
         $this->fields = [];
+        $this->_unlockedFields = [];
```

Comment:
An issue I can see is that if an application has a form like the following, it would break:
```php
$this->Form->unlockField('user_id');
$this->Form->create();
$this->Form->hidden('user_id');
...
$this->Form->end();
```
I am not sure whether the above is correct or not. It might be better to be done in `end()` method instead.